src/reactionsystem_conversions.jl

```diff
@@ -509,13 +523,17 @@ Keyword args and default values:
   ignored. Defaults to the value given when the `ReactionSystem` was constructed (which
   itself defaults to true).
 - `remove_conserved=false`, if set to `true` will calculate conservation laws of the
-  underlying set of reactions (ignoring constraint equations), and then apply them to reduce
-  the number of equations.
+  underlying set of reactions (ignoring coupled ODE or algebraic equations). For each
+  conservation law one steady-state equation is eliminated, and replaced with the
+  conservation law. This ensures a non-singular Jacobian. When using this option, it is
+  recommended to call `ModelingToolkit.structural_simplify` on the converted system to then
+  eliminate the conservation laws from the system equations.
 """
 function Base.convert(::Type{<:NonlinearSystem}, rs::ReactionSystem; name = nameof(rs),
         combinatoric_ratelaws = get_combinatoric_ratelaws(rs),
-        include_zero_odes = true, remove_conserved = false, checks = false,
```

Comment:
There is/was no reason to have `include_zero_odes` for nonlinear systems. It just results in a system equation of the form `0 ~ 0` which is not useful (and is probably indicative of an issue, since if the number of unknowns and equations are only balanced with such an equation it means the system is underdetermined).